enzy_htp/_interface/amber_interface.py

```diff
@@ -2517,14 +2523,26 @@ def antechamber_ncaa_to_moldesc(self,
                           " ALWAYS check and explicit assign it using"
                           " Structure.assign_ncaa_chargespin()")
             raise ValueError
+
+        if ncaa.is_modified():
+            atom_type = "AMBER"
+            gaff_type = atom_type
```

Comment:
The gaff_type should not be changed. Our new parameter will still be a combination of the protein FF and GAFF for the modified side chain